integration/integration_test.go

```diff
@@ -664,10 +665,33 @@ func TestMountsnoop(t *testing.T) {
 	t.Parallel()
 
 	mountsnoopCmd := &command{
-		name:           "StartMountsnoopGadget",
-		cmd:            fmt.Sprintf("$KUBECTL_GADGET trace mount -n %s", ns),
-		expectedRegexp: fmt.Sprintf(`%s\s+test-pod\s+test-pod+\s+mount\s+\d+\s+\d+\s+\d+\s+mount\("/mnt", "/mnt", .*\) = -2`, ns),
-		startAndStop:   true,
+		name:         "StartMountsnoopGadget",
+		cmd:          fmt.Sprintf("$KUBECTL_GADGET trace mount -n %s -o json", ns),
+		startAndStop: true,
+		expectedOutputFn: func(output string) error {
+			expectedEntry := &mountTypes.Event{
+				Event:     buildBaseEvent(ns),
+				Comm:      "mount",
+				Operation: "mount",
+				Retval:    -2,
+				Source:    "/mnt",
+				Target:    "/mnt",
+			}
+
+			normalize := func(e *mountTypes.Event) {
+				e.Node = ""
+				e.Pid = 0
+				e.Tid = 0
+				e.MountNsID = 0
+				e.Latency = 0
+				e.Fs = ""
+				e.Data = ""
+				e.Flags = nil
```

Comment:
I noticed in the output that we `Flags` is always `["MS_SILENT"]` (`mount` command seems to use it by default). I wonder if we should also verify it, but maybe it is better to specify the option `-o silent` to be sure it will always be present.

```bash
busyboxPodRepeatCommand(ns, "mount /mnt /mnt -o silent"),
```

Anyway, we can manage it in another PR.